src/addrman.cpp

```diff
@@ -246,12 +246,17 @@ void AddrManImpl::Unserialize(Stream& s_)
 
     uint8_t compat;
     s >> compat;
+    if (compat < INCOMPATIBILITY_BASE) {
+        throw std::ios_base::failure(strprintf("Corrupted addrman database: The compat value (%u) "
+                                               "is lower than the expected %u.",
```

Comment:
INCOMPATIBILITY_BASE isn't the expected compat value but a lower bound: expected are values >= INCOMPATIBILITY_BASE.

Reply:
Ok, I can see how this can be misinterpreted.

What about:

```suggestion
                                               "is lower than the expected minimum value %u.",
```

Reply:
Yes, I'd prefer that,

Reply:
thanks, done